js/test/input-mask.test.js

```diff
@@ -0,0 +1,199 @@
+    const initialCursorPosition = await captureCursorPosition();
+
+    //explicitly move the mouse and click
+    await page.mouse.move(middleX, middleY);
+    await page.mouse.click(middleX, middleY);
+
+    // get the cursor position ofter click, the assumption is after mouse move, an onclick the cursor doesn't Move
+    const cursorPosition = await captureCursorPosition();
+
+    expect(cursorPosition.x).toBeCloseTo(initialCursorPosition.x, 2);
+    expect(cursorPosition.y).toBeCloseTo(initialCursorPosition.y, 2);
+  });
+
+  // Mouse Selection
+  it("should select characters in the Tel Input field - Mouse", async () => {
+    const page = await desktopBrowser.newPage();
+    await installMouseHelper(page);
+    await page.goto(`${config.BASE_URL}/input-mask.php`, {
+      waitUntil: "domcontentloaded",
+    });
+    const telephoneInput = "#tel";
+    const expectedSelection = "12345";
+    await page.type(telephoneInput, "1234567890");
+    // bounding box of input
+    const inputBoundingBox = await page.$eval(telephoneInput, (input) => {
+      const { x, y, width, height } = input.getBoundingClientRect();
+      return { x, y, width, height };
+    });
+    //Click the input and move to the start of the field
+    await page.click(telephoneInput);
+    await page.mouse.move(inputBoundingBox.x, inputBoundingBox.y);
+    // Select using
+    await page.mouse.down();
+    await page.mouse.move(inputBoundingBox.x + 50, inputBoundingBox.y);
+    await page.mouse.up();
+    //get the selection
+    const selectedText = await page.$eval(
+      "span.enable-input-mask__mask-pre-val",
+      (span) => {
+        return window.getSelection().toString();
+      }
+    );
+    // check screenshot for the selection, should match up - "123-45"
+    await page.screenshot({ path: "mouseSelection.png" });
+    expect(expectedSelection).toBe(selectedText);
+  });
+
+  // Keyboard Selection
+  it("should select characters in the Tel Input field - Keyboard", async () => {
+    const page = await desktopBrowser.newPage();
+    await installMouseHelper(page);
+    await page.goto(`${config.BASE_URL}/input-mask.php`, {
+      waitUntil: "domcontentloaded",
+    });
+    const telephoneInput = "#tel";
+    const expectedSelection = "67890";
+    //Focus on the  input and type
+    await page.focus(telephoneInput);
+    await page.type(telephoneInput, "1234567890");
+    await page.keyboard.down("Shift");
+    // Select using keyboard
+    await page.keyboard.press("ArrowLeft", { shift: true });
+    await page.keyboard.press("ArrowLeft", { shift: true });
+    await page.keyboard.press("ArrowLeft", { shift: true });
+    await page.keyboard.press("ArrowLeft", { shift: true });
+    await page.keyboard.press("ArrowLeft", { shift: true });
+    await page.keyboard.up("Shift");
+    //get the selection
+    const selectedText = await page.$eval(
+      "span.enable-input-mask__mask-pre-val",
+      (span) => {
```

Comment:
nit: Since 'span' isn't used, we shouldn't define it
```suggestion
      () => {
```